omod-common/src/test/java/org/mockingbird/test/rest/resource/BirdResource_1_9.java

```diff
@@ -60,6 +61,21 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		return null;
 	}
 	
+	@Override
+	public Model getGETModel(Representation representation) {
+		return null;
+	}
+	
```

Comment:
Interface: https://github.com/gayanW/openmrs-module-webservices.rest/blob/270edab5222fdfd7b8a77adf209e6d280c1688af/omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingResourceHandler.java#L93-L97

Almost all resources inherit: https://github.com/gayanW/openmrs-module-webservices.rest/blob/270edab5222fdfd7b8a77adf209e6d280c1688af/omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/BaseDelegatingResource.java#L76-L101

Yeah I just saw a minor improvement that could be done. I'll push it with the added doc.